doc/sources/guide/packaging-ios.rst

```diff
@@ -56,11 +53,11 @@ Before proceeding to the next step, ensure your application entry point is a fil
 named `main.py`.
 
 We provide a script that creates an initial Xcode project to start with. In the
-command line below, replace `test` with your project name. It must be a
+command line below, replace `title` with your project name. It must be a
 name without any spaces or illegal characters::
 
-    $ ./toolchain.py create <title> <app_directory>
-    $ ./toolchain.py create Touchtracer ~/code/kivy/examples/demo/touchtracer
+    $ .toolchain create <title> <app_directory>
```

Comment:
did you mean to leave the dot prefix here? (and on next line)

Reply:
Nope I messed up when deleting the `./` thanks for catching it.
I've updated the PR with the fix